pkgs/applications/audio/friture/default.nix

```diff
@@ -30,12 +30,23 @@ in py.buildPythonApplication rec {
 
   patches = [
     ./unlock_constraints.patch
+    ./fix_desktop_item.patch
```

Comment:
rather than a patch, how about using `substituteInPlace` ?
IMO it's a littlle more transparent what is happening and also it's more robust in case the file changes (even though this particular file has a low probability of changing).

Reply:
yes, you're right, this makes it a lot more apparent what's happening to someone just reading `default.nix`.